scripts/artifacts/README.md

```diff
@@ -0,0 +1,33 @@
+# Uploading artifacts to a private registry
```

Comment:
lets create two directories `generate-artifacts-manifest` and `upload-artifacts` and group the code, requirements and associated files in these directories. right now its haphazard